### PR TITLE
Bugfix/err not in status@4.0.x

### PR DESCRIPTION
**Pull request by abouteiller**
Issue #9032, port of #9128 from master

### CODE REVIEW

**Review by jsquyres — APPROVED** [review]

**Comment by gpaulsen** [comment]
Lets wait for additional reviews on #9184 before we merge this.

**Comment by janjust** [comment]
Reviewed by @brminich in master and v5.0 - it's good to go

**Pull request merged**
